app/controllers/stats_helpers/donations.rb

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
This is really hard to parse. If we can do this in AR it would be better — but if we have to do this via queries are there any other things we can do to construct queries vs writing them like this? At least you prepared statements? The way it's written now there is no way to recycle this

Reply:
this statement is a recycle of https://data.heroku.com/dataclips/imjqsyuczmvccxmfsogcrnuzicro / templates